modules/gateway/persist.go

```diff
@@ -1,14 +1,19 @@
 package gateway
 
 import (
-	"log"
-	"os"
 	"path/filepath"
 
 	"github.com/NebulousLabs/Sia/modules"
 	"github.com/NebulousLabs/Sia/persist"
 )
 
+const (
+	// nodesFile is the name of the file that contains all seen nodes.
+	nodesFile = "nodes.json"
+	// logFile is the name of the log file.
+	logFile = modules.GatewayDir + ".log"
+)
+
 var persistMetadata = persist.Metadata{
 	Header:  "Sia Node List",
 	Version: "0.3.3",
```

Comment:
This version number is only for the persist file. It should only be changed if the layout of the persist file is changed.